src/pages/en/guides/styling.md

```diff
@@ -26,15 +26,18 @@ Astro `<style>` CSS rules are automatically **scoped by default**. Scoped styles
 ```diff
 <style>
 -  h1 { color: red; }
-+  h1.astro-HHNQFKH6 { color: red; }
++  h1:where(.astro-HHNQFKH6) { color: red; }
 -  .text { color: blue; }
-+  .text.astro-HHNQFKH6 { color: blue; }
++  .text:where(.astro-HHNQFKH6) { color: blue; }
 </style>
 ```
 
-Scopes styles don't leak and won't impact the rest of your site. In Astro, it is okay to use low-specificity selectors like `h1 {}` or `p {}` because they will be compiled with scopes in the final output.
+Scopes styles don't leak and won't impact the rest of your site. In Astro, it is okay to use low-specificity selectors like `h1 {}` or `p {}` because they will be compiled with scopes in the final output. 
```

Comment:
```suggestion
Scoped styles don't leak and won't impact the rest of your site. In Astro, it is okay to use low-specificity selectors like `h1 {}` or `p {}` because they will be compiled with scopes in the final output.
```

Reply:
You found a typo that's been there the whole time!  😬 💜